apps/extension/src/routes/page/onboarding/height.tsx

```diff
@@ -0,0 +1,73 @@
+import { BackIcon } from '@repo/ui/components/ui/icons/back-icon';
+import { Button } from '@repo/ui/components/ui/button';
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from '@repo/ui/components/ui/card';
+import { FadeTransition } from '@repo/ui/components/ui/fade-transition';
+import { usePageNav } from '../../../utils/navigate';
+import { PagePath } from '../paths';
+import { FormEvent, MouseEvent, useState } from 'react';
+import { Input } from '@repo/ui/components/ui/input';
+import { localExtStorage } from '../../../storage/local';
+
+export const ImportWalletCreationHeight = () => {
+  const navigate = usePageNav();
+  const [blockHeight, setBlockHeight] = useState('');
+
+  const handleSubmit = async (event: MouseEvent | FormEvent) => {
+    event.preventDefault();
+
+    if (blockHeight) {
+      // Save the block height to local extension storage
+      await localExtStorage.set('walletCreationBlockHeight', Number(blockHeight));
+    }
+
+    navigate(PagePath.SET_PASSWORD);
+  };
+
+  const handleClick = (event: MouseEvent | FormEvent) => {
+    handleSubmit(event).catch((error: unknown) => {
+      if (error instanceof Error) {
+        console.error('Error during submission:', error.message);
+      } else {
+        console.error('Unexpected error:', error);
+      }
+    });
+  };
+
+  return (
+    <FadeTransition>
+      <BackIcon className='float-left mb-4' onClick={() => navigate(-1)} />
+      <Card className='w-[600px] p-6' gradient>
+        <CardHeader className='items-center'>
+          <CardTitle className='font-semibold'>
+            Enter Block Height When the Wallet Was Created (Optional)
```

Comment:
question: is this wording similar to zcash's? Would it be more accurate to say "Enter the block height of your first Penumbra transaction?". Not sure.

suggestion: Let's just capitalize the first word

Reply:
"Enter the block height of your first Penumbra transaction" is inherently more involved than just simply using the term 'birthday height,' which is a more intuitive concept.

Maybe we should say something like `"Enter the Block Height at the time your wallet was created. This is your wallet's birthday height (Optional)."`?

Reply:
<img width="505" alt="Screenshot 2024-09-01 at 8 41 08 PM" src="https://github.com/user-attachments/assets/f56e13c9-bd35-4a21-b03c-e5f399d8746d">